structopt-derive/src/parse.rs

```diff
@@ -28,13 +28,13 @@ impl Parse for StructOptAttributes {
 
 pub enum StructOptAttr {
     // single-identifier attributes
-    Short(Ident),
-    Long(Ident),
     Flatten(Ident),
     Subcommand(Ident),
     NoVersion(Ident),
 
     // ident [= "string literal"]
+    Short(Ident, Option<LitStr>),
+    Long(Ident, Option<LitStr>),
```

Comment:
Converting upfront would not need this change

Reply:
In this case `ling = "..."` would count as a raw method but we still need to apply `unraw` to it. If you think searching for it in raw methods and applying the fix after-the-fact would be simpler/better - well, you better think again :)